src/Generator/InterfaceGenerator.php

```diff
@@ -0,0 +1,733 @@
+<?php
+
+namespace Zend\Code\Generator;
+
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @author    Daan Biesterbos <daanbiesterbos@gmail.com>
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
```

Comment:
license should be on top of namespace like others. @author should be before class